api/features/versioning/migrations/0007_add_phased_rollout.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 4.2.22 on 2025-08-06 02:31
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("release_pipelines_core", "0002_add_phased_rollout"),
+        ("feature_versioning", "0006_add_pipeline_stage_to_envfeatureversion"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="environmentfeatureversion",
+            name="phased_rollout_state",
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="environment_feature_versions",
```

Comment:
Shouldn't it be a one to one relation as we update the phase_rollout_state and it is linked to only one EFV (I have a slight doubt on the latter)

Reply:
Yeah, good shout! I have updated both segment and environment feature version to `OneToOneField`